### ISSUE TITLE
Add Show Now Playing

### ISSUE THREAD

**Issue by dukkha77** [reupen/album_list_panel#32]
When playing track in Album list panel and seeing other folders, show now playing function doesn't work.(Same ELplaylist, Playlist view) All of "Activate now playing", "Show now playing in playlist" doesn't work.
I have checked Auto-send to playlist.

Elole theme solve this problem that Library panel makes "Library Playback", "Library Selection" playlists.

Am I missing something?

**Comment by reupen** [comment]
No, you're not missing anything. Two separate playlists is a similar solution to what the Filter panel uses. 

It looks like adding support for the native library viewer selection playlist would also allow it to work if you select something else in Album list panel and then return to the original node (but not while looking at other things in Album list panel).

Which of these scenarios is closer to what you're trying to do?

A:

1. Select and play album A in Album list panel
2. Select album B in Album list panel
3. Use the Show now playing command

B:

1. Select and play album A in Album list panel
2. Select album B in Album list panel
3. Select album A in Album list panel
4. Use the Show now playing command

**Comment by dukkha77** [comment]
Thanks for quick answer.
I just want A. but I tried B, B is not work too.
I have tried some scenarios

1. Selcet and play album A in Playlist Switcher, then Select album B in Playlist Switcher, Use the Show now playing command
2. Selcet and play album A in Playlist Switcher, then Select album B in Album list panel, Use the Show now playing command
3. Select and play album A in Album list panel, then Select album B in Album list panel, Use the Show now playing command

1, 2 is OK. 
3 is not work.
